packages/yoastseo/src/languageProcessing/researches/getKeywordDensity.js

```diff
@@ -26,7 +21,22 @@ export default function( paper, researcher ) {
 		return 0;
 	}
 
-	const keywordCount = researcher.getResearch( "keywordCount" );
+	const keyphraseCount = researcher.getResearch( "keyphraseCount" );
 
-	return ( keywordCount.count / wordCount ) * 100;
+	return ( keyphraseCount.count / wordCount ) * 100;
+}
+
+/**
+ * Calculates the keyphrase density.
+ *
+ * @deprecated Since version 20.12. Use getKeyphraseDensity instead.
```

Comment:
The version here could change depending on our plan to ship this change

Reply:
Given that we're currently not releasing `yoastseo`, let's not at a version number here. Once we release `yoastseo` we can check all `@deprecated` tags and add/update the version number.